modules/plugins/statusline/lualine/config.nix

```diff
@@ -11,7 +11,7 @@
   inherit (lib.generators) mkLuaInline;
 
   cfg = config.vim.statusline.lualine;
-  breadcrumbsCfg = config.vim.ui.breadcrumbs;
+  bcfg = config.vim.ui.breadcrumbs;
```

Comment:
Highly meh naming

Reply:
highly doesn't matter :trollface: 

what should it be? breadcrumbsCfg is way too long

Reply:
Keep bcfg I guess, but at least capitalize the C?

Reply:
done